utils/notifications.js

```diff
@@ -401,20 +401,31 @@ async function getParticipantsAndSendNotifications({ notificationSpec, cutoffTim
 
       try {
         await sgMail.send(emailBatch);
+        await saveNotificationBatch(emailRecordArray);
+        emailCount += emailRecordArray.length;
       } catch (error) {
-        console.error(`Error sending emails for ${notificationSpec.id}(${readableSpecString}).`, error);
+        if (error.message.startsWith("saveNotificationBatch")) {
+          console.error(`Error saving email records for ${notificationSpec.id}(${readableSpecString}).`, error);
+        } else {
+          console.error(`Error sending emails for ${notificationSpec.id}(${readableSpecString}).`, error);
```

Comment:
Will these be available in DataDog or only in GCP?

Reply:
GCP logs are imported to Datadog.